data/coalition/coalition weapons.txt

```diff
@@ -402,3 +402,42 @@ effect "ion rain impact"
 	"random spin" 2
 	"random velocity" 10
 	"velocity scale" -.1
+
+
+
+outfit "MCS Extractor"
+	category "Guns"
+	licenses
+		Coalition
+	cost 106000
+	thumbnail "outfit/mcs extractor"
+	"mass" 18
+	"outfit space" -18
+	"weapon capacity" -18
+	"asteroid scan power" 18
+	"gun ports" -1
+	weapon
+		sound "drill"
+		"hit effect" "drilling"
+		"velocity" 10
+		"lifetime" 1
+		"reload" 1
+		"firing energy" .78
+		"firing heat" .68
+		"shield damage" .2
+		"hull damage" 1.4
+		"minable damage" 8.2
+	description `Not legally a weapon under Heliarch definitions, the "Mineral Collection and Sorting" extraction unit consists of a set of sharp, durable blades designed to quickly break down mineable asteroids. It also comes with an integrated scanning system that looks for asteroids rich in minerals, to facilitate the mining efforts.`
```

Comment:
```suggestion
	description `Not legally a weapon under Heliarch definitions, the "Mineral Collection and Sorting" extraction unit consists of a set of sharp, durable blades designed to quickly break down minable asteroids. It also comes with an integrated scanning system that looks for asteroids rich in minerals to facilitate the mining efforts.`
```
While mineable and minable are both correct, the game consistently uses minable.